Sources/MongoSwift/APM.swift

```diff
@@ -1,14 +1,93 @@
 import CLibMongoC
 import Foundation
 
-/// A protocol for monitoring events to implement, specifying their name.
-public protocol MongoEvent {
-    /// The name this event will be posted under.
-    static var eventName: Notification.Name { get }
+/// A protocol for `CommandEvent` handlers to implement.
+public protocol CommandEventHandler: AnyObject {
```

Comment:
These now have class requirements so they can be stored `weak`. I think it probably makes sense that the handlers are always reference types anyways.